DashWallet/Sources/Categories/DSTransaction+DashWallet.swift

```diff
@@ -85,10 +85,7 @@ extension DSTransaction {
     }
 
     var direction: DSTransactionDirection {
-        let currentAccount = DWEnvironment.sharedInstance().currentAccount
-        let account = accounts.contains(where: { ($0 as! DSAccount) == currentAccount }) ? currentAccount : nil
-
-        return account != nil ? chain.direction(of: self) : .notAccountFunds
```

Comment:
This doesn't seem necessary as `chain.direction` checks all accounts anyway and returns 0 for unrelated accounts.